version

```diff
@@ -1 +1 @@
-0.6.4
+0.6.5
```

Comment:
I missed that this change was made on this initial PR's commit. Would you mind getting it reverted @tiny-dancer? Pending that I see no reason we can't get this merged in. (FWIW, I am intending for a few other changes to go out in the 0.6.5 release that is tentatively slated for Oct. 1st.)